Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/NodePublishing/PublishMovedNodesWithoutDimensions.feature

```diff
@@ -82,7 +82,7 @@ Feature: Publishing moved nodes without dimensions
       | nodesToPublish           | [{"contentStreamId": "user-cs-identifier", "dimensionSpacePoint": {}, "nodeAggregateId": "sir-david-nodenborough"}] |
     And the graph projection is fully up to date
 
-    Then I expect the graph projection to consist of exactly 4 nodes
+    Then I expect the graph projection to consist of exactly 6 nodes
```

Comment:
I had to adjust the expected number of nodes here and in 3 cases below because moving a node now might "materialize" it in order to update the `lastModified` timestamp per affected content dimension

Reply:
We agreed that node moving (and en/disabling) does not update the last modified timestamp, so this is no longer an issue